### DISCUSSION TITLE
Passive scan only works when setting (active) scan on

### DISCUSSION THREAD

**Discussion by locnide** [bluez/bluer#153] [Q&A]
Hello, 

When I run the example `le_passive_scan`, it never passively reads advertisements.
But when I set the bluetooth (active) scan on, it reads advertisements (it is not a filter issue)

This seems to be a bluez issue as it also does it when I activate passive scan using `bluetoothctl`.

Is bluez passive scan implementation only working with specific adapters ? If yes, how do you know which adapters are compatible ?
Or is it something else ? Did anyone have the same issue with passive scan ?

**Comment by surban** [comment]
I recommend asking on the BlueZ Slack or mailing list, since this issue is not specific to BlueR.
